multicluster/apis/multicluster/v1alpha1/multiclusterset_types.go

```diff
@@ -0,0 +1,113 @@
+/*
+Copyright 2021 Antrea Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+type ClusterStatus string
+
+const (
+	ClusterStatusReachable   ClusterStatus = "Reachable"
+	ClusterStatusUnreachable ClusterStatus = "Unreachable"
+)
+
+// MemberCluster defines member cluster information.
+type MemberCluster struct {
+	// Identify member cluster in multi-cluster set.
+	ClusterID string `json:"clusterID,omitempty"`
+	// API server of the member or the leader cluster.
```

Comment:
A little confusion here, I believe you mean it's the API server of the member cluster itself, but a member cluster can also be a leader cluster?

Reply:
yes, should be API Server if the leader cluster.

Reply:
done.